Dockerfile

```
@@ -0,0 +1,11 @@
+FROM alpine:latest
+RUN apk add --no-cache --update bash nodejs npm
+
+WORKDIR /var/www
+COPY . .
+
+RUN npm install
+RUN npm run build
```

Лучше перед установкой модулей закешировать этот слой, чтобы каждый раз не ставить зависимости, если они не изменились. Подробнее можно почитать здесь https://docs.docker.com/get-started/09_image_best/#layer-caching
```suggestion
WORKDIR /var/www
COPY package*.json .
RUN npm install
COPY . .
RUN npm run build
```